components/script_traits/script_msg.rs

```diff
@@ -85,7 +85,7 @@ pub enum ScriptMsg {
     LoadUrl(PipelineId, LoadData, bool),
     /// Dispatch a mozbrowser event to a given iframe,
     /// or to the window if no subpage id is provided.
-    /// First PipelineId is for the parent, second PipelineId is for the actual pipeline.
+    /// First PipelineId is for the parent, second PipelineId is in the frame.
```

Comment:
What does it mean that it is "in the frame"? It still lives here so it's a but confusing.